search-commons/src/main/java/no/unit/nva/search2/model/IParameterKey.java

```diff
@@ -0,0 +1,65 @@
+package no.unit.nva.search2.model;
+
+import java.net.URLDecoder;
+import java.nio.charset.StandardCharsets;
+import java.util.Collection;
+import java.util.function.Predicate;
+
+public interface IParameterKey {
```

Comment:
This little interface makes enums into usefull classes.
if we need other querybuilders, they will work with a new enums "out of the box"